examples/production/smart_contracts/hello_world/contract.algo.ts

```diff
@@ -2,10 +2,6 @@ import { Contract } from '@algorandfoundation/algorand-typescript'
 
 export class HelloWorld extends Contract {
   public hello(name: string): string {
```

Comment:
I think we wanted to take out `public` as well so it is as clean as possible

Reply:
Fixed - remove the eslint rule as well.